### PR TITLE
feat: add an EMPTY ecosystem for withdrawn records

### PR DESCRIPTION
**Pull request by hogo6002**
Some records will be withdrawn from upstream and may also have their affected package information deleted. These records currently cannot be exported and may cause confusion for our customers due to missing records.  Adding an "EMPTY" ecosystem will represent these records for exporting.

### CODE REVIEW

**Comment by another-rex** [comment]
It's maybe even worth adding square brackets around `[Empty]` just to make sure that it will never be a real ecosystem in the future.

**Review by another-rex — APPROVED** [review]
LGTM, though let's wait till after this week's release before merging this in.

**Review by cuixq — APPROVED** [review]

**Pull request merged**
